orangecontrib/network/widgets/graphview.py

```diff
@@ -252,6 +257,10 @@ def __init__(self, parent=None):
         self.positionsChanged.connect(self._update_positions, type=Qt.BlockingQueuedConnection)
         self.animationFinished.connect(self.finish)
 
+    def _cursor_change(self, cursor):
+        self._cursor.setShape(cursor)
+        self.setCursor(self._cursor)
```

Comment:
Where else is `self._cursor` used? This method is redundant. Just `.setCursor()` directly.

Reply:
Great, thanks!